# HG changeset patch
# User Rik <rik@octave.org>
# Date 1523311154 25200
#      Mon Apr 09 14:59:14 2018 -0700
# Node ID d2558ba799d8956e2b797392b53545b815458008
# Parent  ecc37eef11ccfa910bf961510d5a5407392d9c56
Change the GUI Import/Export Shortcuts strings to end with ellipsis.

* shortcut-manager.cc (do_import_export): Change GUI strings to end with "file...".

diff --git a/libgui/src/shortcut-manager.cc b/libgui/src/shortcut-manager.cc
--- a/libgui/src/shortcut-manager.cc
+++ b/libgui/src/shortcut-manager.cc
@@ -622,22 +622,22 @@ namespace octave
     // get the filename to read or write the shortcuts,
     // the default extension is .osc (octave shortcuts)
     if (action != OSC_DEFAULT)
       {
         QString file;
 
         if (action == OSC_IMPORT)
           file = QFileDialog::getOpenFileName (this,
-                                               tr ("Import shortcuts from file"), QString (),
+                                               tr ("Import shortcuts from file..."), QString (),
                                                tr ("Octave Shortcut Files (*.osc);;All Files (*)"),
                                                nullptr, QFileDialog::DontUseNativeDialog);
         else if (action == OSC_EXPORT)
           file = QFileDialog::getSaveFileName (this,
-                                               tr ("Export shortcuts to file"), QString (),
+                                               tr ("Export shortcuts to file..."), QString (),
                                                tr ("Octave Shortcut Files (*.osc);;All Files (*)"),
                                                nullptr, QFileDialog::DontUseNativeDialog);
 
         if (file.isEmpty ())
           return false;
 
         QSettings *osc_settings = new QSettings (file, QSettings::IniFormat);
 
